router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala

```diff
@@ -10,29 +10,30 @@ import com.twitter.finagle.tracing.Trace
 object TracingFilter extends SimpleFilter[Request, Response] {
 
   def apply(req: Request, service: Service[Request, Response]) = {
+    Trace.recordRpc(req.method.toString)
     Trace.recordBinary("http.method", req.method.toString)
```

Comment:
I'm pretty ambivalent on this. Finagle's http codec records "http.uri", so I'd like to keep that one for consistency.  I could go either way on the rest of them.  I also noticed that Finagle's annotations tend to be hierarchical, so we should probably use "http.req" and "http.rsp", instead of "req.http" and "rsp.http".

So that said, how do you feel about this as the set of annotations that we record:

* http.uri
* http.req.method
* http.req.host
* http.req.version
* http.req.content-length
* http.req.content-type
* http.req.transfer-encoding
* http.rsp.version
* http.rsp.content-length
* http.rsp.content-type
* http.rsp.transfer-encoding
* http.rsp.status